transport/layer.py

```diff
@@ -63,3 +63,9 @@ def construct_light_clients_nodes(self, config):
          This function must return a list of objects that are subtype of TransportNode and correspond to light client
          transport nodes
         """
+
+    @abstractmethod
+    def get_light_client_transport(self, address) -> TransportNode:
```

Comment:
since this returns a _node_, i would maybe call this `get_light_client_node` or `get_light_client_transport_node`.

as usual with my name suggestions: not a blocker

Reply:
changed to `get_light_client_transport_node`